tools/bowtie2/main.nf

```diff
@@ -0,0 +1,55 @@
+nextflow.preview.dsl=2
+params.genome = ''
+
+process BOWTIE2 {
+	// depending on the genome used one might want/need to adjust the memory settings.
+	// For the E. coli test data this is probably not required
+
+	// label 'bigMem' 
+	// label 'multiCore'
+		
+    input:
+	    tuple val(name), path(reads)
+		val (outdir)
+		val (bowtie2_args)
+		val (verbose)
+
+	output:
+	    path "*bam",  emit: bam
+		path "*stats.txt", emit: stats 
+
+	publishDir "$outdir/bowtie2",
+		mode: "copy", overwrite: true
```

Comment:
![tabs spaces](https://external-preview.redd.it/EcwKGjcxeR77frd8nFeG2ggEpCSLtGbudsABDPGBTe8.png?auto=webp&s=738e71260d3ec2e1f8ec863ad3f1de9b6aff51ad)

Reply:
Ahh, thats's why it looks like that, in VS Code it all looks fine. I thought the whole indentation stuff is handled by the editors these days? Maybe not...

Reply:
It should be a preference. I have plugins in atom to convert tabs to spaces and to highlight tabs, so it’s nice and clear and easy to stick to just spaces. It also highlights and auto-cleans trailing whitespace. The only downside is if you save a messy file then the diff shows that you edited half of the lines even if you only changed one line 🤭